.idea/cluster.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess you have add that .idea/ directory before update the ``.gitignore`` could you remove them ?

> **Advice**: You could add a ``~/.gitignore`` in your home directory to ignore such directory
> related to your laptop it should be then ignore in all your projects!

Reply:
.idea files removed !